client/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './Footer.module.scss';
+import logoImage from '../../assets/image/logo.png';
+interface IProps {}
+
+export const Footer: React.FC<IProps> = ({}) => {
+    const renderLogo = () => {
+        return (
```

Comment:
Looks like you already have logo in Header. So Logo can be a separate reusable component.